nixos/modules/programs/gns3.nix

```diff
@@ -0,0 +1,110 @@
+{ pkgs, config, lib, ... }:
+
+with lib;
```

Comment:
```suggestion
```

Reply:
That's necessary for the `mkOption`, `types.XXX`, `mkDoc`, etc.

Reply:
No, it is not.
Prefix them or write an explicit `inherit (lib)` listing.

Reply:
Done in the last [rebase](https://github.com/NixOS/nixpkgs/compare/ea492c6e07d5d0444982949791ea4f58cf65f071..27f62184229e9e6c505d49e5ee95ef562477598a)